Assets/Scripts/SS3D/Systems/Inventory/Items/ItemActor.cs

```diff
@@ -26,62 +26,53 @@ namespace SS3D.Systems.Inventory.Items
     [RequireComponent(typeof(Rigidbody))]
     [RequireComponent(typeof(NetworkTransform))]
     [RequiredLayer("Items")]
-    public class Item : InteractionSource, IInteractionTarget
+    public class ItemActor : InteractionSource, IInteractionTarget
     {
-        [FormerlySerializedAs("_itemId")]
-        [FormerlySerializedAs("_itemIdID")]
-        [FormerlySerializedAs("ItemID")]
+        #region ItemActor
         [Header("Item settings")]
-        [SerializeField]
-        [HideInInspector]
-        public Data.Enums.ItemId _itemId;
 
-        [SerializeField] private string _name;
+        [FormerlySerializedAs("Name")]
+        [SerializeField] private string _startingName;
 
-        [SerializeField] private Sprite _sprite;
+        [FormerlySerializedAs("Weight")]
+        [SerializeField] private float _startingWeight;
 
-        [SerializeField] private Rigidbody _rigidbody;
-
-        public string Name => _name;
+        [FormerlySerializedAs("Traits")]
+        [SerializeField] private List<Trait> _startingTraits;
 
-        /// <summary>
-        /// The item's relative weight in kilograms.
-        /// </summary>
-        [SerializeField] private float _weight;
+        [SerializeField] private Rigidbody _rigidbody;
 
         [Tooltip("the item prefab, you can click on the item name and drag from Unity's file explorer")]
         public GameObject Prefab;
+
         [Header("Attachment settings")]
+
         [Tooltip("a point we use to know how the item should be oriented when held in a hand")]
         public Transform AttachmentPoint;
+
         [Tooltip("same point but for the left hand, in cases where it's needed")]
         public Transform AttachmentPointAlt;
-        [Tooltip("The size of the item inside a container")]
-        private Vector2Int _size;
-        private Container _container;
 
-        public Vector2Int Size => _size;
+        private Item item = new Item();
 
-        public List<Trait> traits;
+        public string Name => item.Name;
+        public ItemId ItemId => item.ItemId;
+        public Vector2Int Size => item.Size;
+        public List<Trait> traits => item.Traits;
```

Comment:
replace that with 
```
[SyncObject]
        private readonly SyncList<Trait> traits = new();
```

add the property : 

`public ReadOnlyCollection<Trait> Traits => ((List<Trait>) traits.Collection).AsReadOnly();`